grafana-backup/run.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -l
+# shellcheck disable=SC1090
+
+# This script copies Grafana dashboard jsons, tar them and put into HDFS and EOS folder.
+# Ref for alerting: https://github.com/dmwm/CMSSpark/blob/master/bin/cron4aggregation
+
+##H Usage: run.sh KERBEROS_KEYTAB TOKEN_LOCATION HDFS_PATH FILESYSTEM_PATH
+##H Example:
+##H        run.sh keytab keys/token.json /cms/backups/grafana/ /eos/cms/store/group/offcomp_monit/
+
+# help definition
+if [ "$1" == "-h" ] || [ "$1" == "-help" ] || [ "$1" == "--help" ] || [ "$1" == "help" ] || [ "$1" == "" ]; then
+    grep "^##H" <"$0" | sed -e "s,##H,,g"
+    exit 1
+fi
+
+# Get utils functions
+. "${script_dir}"/../scripts/utils.sh
+
+# Authenticate with Kerberos keytab
+util_kerberos_auth_with_keytab $1
+
+# Change working directory
+cd "$(dirname "$0")" || exit
+
+addr=ceyhun.uzunoglu@cern.ch
+
+# Add cvmfs envs to run amtool
+export VO_CMS_SW_DIR=/cvmfs/cms.cern.ch
+source $VO_CMS_SW_DIR/cmsset_default.sh
```

Comment:
What needs to be done with this part? Do we need to add these scripts separately to the repo since we are moving to Kubernetes?

Reply:
We don't need cvmfs which host the pre-built executables. We'll build these executables(amtool, etc.) and use them in docker image.